srcpkgs/mangadesk/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'mangadesk'
+pkgname=mangadesk
+version=0.7.8
+revision=1
+archs="x86_64"
```

Comment:
Why is this limited to `x86_64`?

Reply:
Hi, I limited it to `x86_64` to mimick what is being done on other distributions' mangadesk package.

It seems to cross build (armv7l, armv6l-musl, aarch64-musl) alright on my machine, though.
I guess I could remove the `archs` line and the `depends` line as well.

Reply:
yes, the depends is also wrong, it would depend on `glibc` not `glibc-devel`, but even then xbps-src detects linked shared libary dependencies on it own.

Reply:
Force pushed a new commit to resolve this.